content/content.types.ts

```diff
@@ -1,3 +1,6 @@
+export interface AriaLabels {
+    ctaButton?: string
+}
```

Comment:
And an extra line after this.

Reply:
@e-papanicolas oooff sorry my sloppiness, corrections have been implemented

Reply:
All good!